src/pages/KadraVykhovnykiv/UpdateKadraForm.tsx

```diff
@@ -45,21 +45,21 @@ const UpdateKadraForm: React.FC<FormUpdateKadraProps> = (props: any) => {
       link: record.link,
     };
 
-    await kadrasApi
+    /*await kadrasApi
```

Comment:
Why commented code are here?
If it's really needed, plese leave an appropriate todo comment